{{ cookiecutter.project_slug }}/frontend/Makefile.default

```diff
@@ -52,13 +52,13 @@ format-lint: ## Format Code with Lint
 .PHONY: format
 format: format-prettier format-stylelint format-lint ## Format the codebase according to our standards
 
-.PHONY: i18n
```

Comment:
I think it was ok, why did you remove it?

Reply:
Ordering of Makefile